napari_aicssegmentation/controller/workflow_select_controller.py

```diff
@@ -67,7 +67,11 @@ def unselect_channel(self):
     def select_workflow(self, workflow: str):
         self.model.active_workflow = workflow
         # TODO create Layer 0 -> https://github.com/AllenCell/napari-aicssegmentation/issues/27
-        self.router.workflow_steps()
+        self.viewer.add_image(
+            self.model.selected_layer.data,
+            name="0. " + self.model.layers[0] + ": ch[" + str(self.model.selected_channel.index) + "] " + workflow,
+        )
+        self.router.workflow_steps(workflow)
```

Comment:
Since we have the active workflow available in the model, it doesn't need to be passed in any further. This can be simplified